src/bp/ui-studio/src/web/views/FlowBuilder/sidePanelFlows/index.tsx

```diff
@@ -104,16 +105,14 @@ const SidePanelContent: FC<Props> = props => {
 
 const SidePanelInspectorContent: FC<Props> = props => {
   return (
-    <SidePanel style={{
-      position: 'absolute',
-      right: 0,
-      width: 240,
-      zIndex: 1,
-      paddingTop: 70,
-      ...(!props.showFlowNodeProps && { width: 0 })
-    }}>
-      {props.showFlowNodeProps ? ( <Inspector /> ) : null}
-    </SidePanel>
+    <div className={props.showFlowNodeProps ? style.rightPanelActive : style.rightPanel}>
+      <SidePanel>
+        <SidePanelSection label="Inspector">
+          
```

Comment:
Please make sure you have the ESLint extension on VS Code, you can also enable auto fix so files are fixed right when you save them

Reply:
Done thanks for the auto fix feature !